README.md

```diff
@@ -1,23 +1,51 @@
 <img src="http://moveit.ros.org/assets/images/moveit2_logo_black.png" alt="MoveIt! Logo" width="200"/>
 
------
+The MoveIt Motion Planning Framework for ROS 2
 
-Contributions are being reviewed. Refer to https://github.com/acutronicrobotics/moveit2 for a fork maintained by @AcutronicRobotics that includes the latest changes.
+We are currently working on the upcoming Beta version of MoveIt 2.
+More information about our roadmap and future milestones will be added soon.
 
------
+### Build from Source
+
+> Note: Currently, only the moveit\_core packages are being compiled.
+
+These instructions assume you are running on Ubuntu 18.04.
+
+1. [Install ROS2 Dashing](https://index.ros.org/doc/ros2/Installation/Linux-Install-Debians/) (Make sure to set `export CHOOSE_ROS_DISTRO=Dashing` and to source `/opt/ros/dashing/setup.bash`)
+
+1. [Install ROS2 Build Tools](https://index.ros.org/doc/ros2/Installation/Linux-Development-Setup/#install-development-tools-and-ros-tools)
+
+1. Create a colcon workspace:
+
+        export COLCON_WS=~/ws_ros2/
+        mkdir -p $COLCON_WS/src
+        cd $COLCON_WS/src
+
+1. Download the repository and install any dependencies:
+
+        git clone git@github.com:ros-planning/moveit2.git
+        vcs import < moveit2/moveit2.repos
+        rosdep install -r --from-paths . --ignore-src --rosdistro Dashing -y
```

Comment:
`Dashing` should be `dashing`, right .? when I run this it doesn't install some of the dependencies when I replaced it with the lowercase it installed them

Reply:
Great catch!